src/python/pants/engine/fs.py

```diff
@@ -208,6 +208,11 @@ class Workspace:
   """Abstract handle for operations that touch the real local filesystem."""
   _scheduler: "SchedulerSession"
 
+  def materialize_directory(
+    self, directory_to_materialize: DirectoryToMaterialize
+  ) -> MaterializeDirectoryResult:
+    return self._scheduler.materialize_directory(directory_to_materialize)
```

Comment:
Can you add a doc-comment pointing people at the parallel version and encouraging its use?